documentation/internal/Assignment-Process.md

```diff
@@ -0,0 +1,57 @@
+# Assignment Process
+
+The logic of assigning a review spans between both web app and server, front end is only concerned with queries and mutations to review_assignment and review_question_assignment tables, whereas back end does most of the grunt work (through actions and triggers) and generates records in those tables based on user permissions and states of application and it's review.
+
+## Schema
+
+Seems like a good starting (anchor) point for describing this functionality, let's have look:
+
+![Review Assignment](images/Review-Assignment.png)
+
+First of all, some bulletpoint (and rules) to help clarify the diagram
+
+- One review assignment should exists per reviewer/application/stage/level
```

Comment:
```suggestion
- One review assignment should exist per reviewer/application/stage/level
- ```

Reply:
f74632106a6a1af38d301b000f3fb8027bf7178e
did a manual commit, i saw an extra line in the suggested change